src/main/java/io/jenkins/plugins/opentelemetry/job/MonitoringRunListener.java

```diff
@@ -395,6 +423,19 @@ public void _onFinalized(@NonNull Run run) {
             } else {
                 this.runAbortedCounter.add(1);
             }
+
+            String pipelineId =
+                runDurationHistogramAllowList.matcher(run.getParent().getFullName()).matches()
+                    &&
+                    !runDurationHistogramDenyList.matcher(run.getParent().getFullName()).matches() ?
+                    run.getParent().getFullName() : "#other#";
+            runDurationHistogram.record(
+                TimeUnit.SECONDS.convert(run.getDuration(), TimeUnit.MILLISECONDS),
+                Attributes.of(
+                    JenkinsOtelSemanticAttributes.CI_PIPELINE_ID, pipelineId,
+                    JenkinsOtelSemanticAttributes.CI_PIPELINE_RUN_RESULT, result.toString(),
+                    JenkinsOtelSemanticAttributes.CI_PIPELINE_RUN_COMPLETED, result.isCompleteBuild())
```

Comment:
`result.isCompleteBuild()==false` means that the run was aborted.
It makes sense to count the aborted jobs separately. :+1:

However aren't aborted jobs already covered by the `result==Result.ABORTED` (`CI_PIPELINE_RUN_RESULT` attribute)?
Do we need a separate completed attribute?

Reply:
```suggestion
                    // TODO do we need this `completed` dimension that we captured on spans? Can't this inferred from the `result` attribute ?
                    JenkinsOtelSemanticAttributes.CI_PIPELINE_RUN_COMPLETED, result.isCompleteBuild())
```

> However aren't aborted jobs already covered by the result==Result.ABORTED (CI_PIPELINE_RUN_RESULT attribute)?

It was recopied from what we put in the span. Question added a a todo